src/components/page/StandardPageWithSelection.tsx

```diff
@@ -45,36 +45,18 @@ const withRowSelection = <T,>({
   return Enhanced;
 };
 
-const withHeaderSelection = <T,>({
-  canSelect,
-  HeaderMapper,
-  isExpanded,
-  isSelected,
-  toggleSelectFor,
-}) => {
+export function withHeaderSelection<T>({ HeaderMapper, isExpanded }) {
```

Comment:
why changed to function notation instead of arrow?

Reply:
Just leftover from rebasing. Will update.